packages/web3-eth-accounts/src/account.ts

```diff
@@ -131,7 +131,8 @@ export const hashMessage = (message: string): string => {
  * }
  * ```
  */
-export const sign = (data: string, privateKey: Bytes): SignResult => {
+
+export const signWithPrivateKey = (data: string, privateKey: Bytes): SignResult => {
```

Comment:
This will add extra step for users to migrate to 4.x, and we should make it least effort for end users instead of additional steps. so I think this change should be reverted.

Reply:
for solving same method name issue, I think we can create a wrapper here `signWithPrivateKey` for exporting through web3 package and also keep `sign` here for old users least effort.

Reply:
I reverted the renaming.
Thanks,